addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRule.java

```diff
@@ -698,4 +685,29 @@ private static String insertUninitVar(String cmd) {
         return cmd.replaceAll("\\s", Matcher.quoteReplacement(var + " "))
                 .replaceAll("\\/", Matcher.quoteReplacement(var + "/"));
     }
+
+    private AlertBuilder buildAlert(
+            String param, String attack, String evidence, String otherInfo, HttpMessage msg) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(evidence)
+                .setMessage(msg)
+                .setOtherInfo(otherInfo);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(
+                buildAlert(
+                                "qry",
+                                "a;cat /etc/passwd ",
+                                "root:x:0:0",
+                                Constant.messages.getString(
+                                        MESSAGE_PREFIX + "otherinfo." + "feedback-based",
+                                        "a;cat /etc/passwd "),
```

Comment:
Better call `getOtherInfo("feedback-based", "a;cat /etc/passwd ")` (also might be better to extract an enum from/for the `testType`).

Reply:
Do we want two alert examples here? I can do that too just trying to clarify.

Reply:
The comment was just to avoid duplication between the actual and the example alert.